src/js/features/better-viewer-list.js

```diff
@@ -21,12 +22,18 @@ function renderViewerList() {
 
         // Filter empty subsections
         for (var i = 0; i < results.length - 1; i++) {
+            if (!results[i].filter) count += 1;
             if (results[i].filter && results[i + 1].text === ' ') i++;
             else tmpResults.push(results[i]);
         }
         results = tmpResults;
+    } else {
+        count = chatterCount;
     }
     viewList.render(results);
+
+    var label = count + (count < 2 ? ' viewer' : ' viewers');
```

Comment:
The problem is that the list includes headers such as "MODERATORS" or "ADMINS" and I can't predict how many of those there are in the list.

The chatter/viewer thing really bugs me too, the issue is, it's called "viewers list" even though it's supposed to be "chatters list", but yeah I'm stuck between changing everything to chatter, or having it be inconsistent. Both solutions kinda suck.